modules/varnish/templates/default.vcl

```diff
@@ -165,6 +165,7 @@ sub mw_request {
 		req.http.Host == "sslrequest.miraheze.org"
 	) {
 		set req.backend_hint = mwtask111;

```

Comment:
The old mwtask111 here should be removed.

Reply:
Well then don't switch it at all, until the switch is to be made I guess.

Reply:
Shouldn't sslrequest.miraheze.org this be on puppet111 only?